.changeset/many-rockets-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Added the new rule [`useIterableCallbackReturn`](https://biomejs.dev/linter/rules/use-iterable-callback-return), which enforces consistent return values in iterable callbacks.
```

Comment:
When implementing new features, we want to talk a bit more about them. In this particular case, we want to list all the iterable callbacks, and provide at least an example that shows when the rule is triggered

Reply:
Done